core/typedefs.h

```diff
@@ -135,17 +129,16 @@ inline void __swap_tmpl(T &x, T &y) {
 	y = aux;
 }
 
-#endif //swap
+#endif // Swap
 
-/* clang-format off */
+// clang-format off
```

Comment:
I don't mind either way for most comments, but please leave the clang format ones with `/* */` style.